driver-core/src/main/com/mongodb/internal/connection/SslHelper.java

```diff
@@ -51,6 +58,27 @@ public static void enableSni(final String host, final SSLParameters sslParameter
         }
     }
 
+    public static void configureSslSocket(final Socket socket, final SslSettings sslSettings, final InetSocketAddress inetSocketAddress) throws
```

Comment:
This method is copied without changes from [SocketStreamHelper.java#L90](https://github.com/mongodb/mongo-java-driver/blob/80a9cb4f961f727f4a5ac85882e658723e7f04d9/driver-core/src/main/com/mongodb/internal/connection/SocketStreamHelper.java#L90)